runtime/lua/vim/filetype.lua

```diff
@@ -851,7 +851,7 @@ local extension = {
   stm = function() vim.fn["dist#ft#FThtml"]() end,
   tcsh = function() vim.fn["dist#ft#SetFileTypeShell"]("tcsh") end,
   tex = function() vim.fn["dist#ft#FTtex"]() end,
-  tf = function() vim.fn["dist#ft#FTtf"]() end,
+  tf = function(path, bufnr) return require("vim.filetype.detect").tf(path, bufnr) end,
```

Comment:
Could we introduce a local variable `local ft_detect = require("vim.filetype.detect")` for this?

Reply:
Sorry, but no -- this would evaluate the `require` unconditionally for _every_ filetype detection, which would have a noticeable performance regression.